src/binding.rs

```diff
@@ -75,6 +75,16 @@ pub enum Binding {
     ///
     /// Useful for expressing empty bindings in [presets](crate::preset).
     None,
+    /// Any key, mouse button, or gamepad button, captured as [`ActionValue::Bool`].
+    /// If used with a context that selects an input device, such as
```

Comment:
Let's put a newline here. Usually in docs you have a short description and then details in additional paragraphs.
```suggestion
    /// Any key, mouse button, or gamepad button, captured as [`ActionValue::Bool`].
    ///
    /// If used with a context that selects an input device, such as
```